procedures/unit-testing-basics.ipf

```diff
@@ -589,6 +591,26 @@ static Function incrRunCount()
 	run_count +=1
 End
 
+/// Get or create the wave that contains the failed procedures
+static Function/WAVE GetFailedProcWave()
+	string name = "FailedProcWave"
+
+	dfref dfr = GetPackageFolder()
+	WAVE/Z/T wv = dfr:$name
+	if(WaveExists(wv))
+		if(DimSize(wv, 0) >= TC_SUMMARY_INIT_SIZE)
+			return wv
+		else
+			KillWaves wv
```

Comment:
It is unexpected that `wv` will be smaller than `TC_SUMMARY_INIT_SIZE` in any case. I initialize it with this size and after that it can only grow. If it is smaller it is some kind of user error and I thought it would be safer to just kill and recreate it.

Reply:
That is not a pattern we would use often. The reason is that killing a wave and then recreating it, is expensive as it involves memory allocations.

I would actually think that we can just return it, if it exists or? We do reset the index to 0 in TestBegin.

Reply:
I think that we can assume that user code does not change IUTF data in the IUTF package folder.
Thus, for these we just need proper initialization when a UTF run is started and use the wave when it is already there.

(Igor has no feature to protect "our" package folder from changes by user code)